src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -1658,7 +1661,10 @@ public static void Xml_TypeWithTimeSpanProperty()
 <TypeWithTimeSpanProperty xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns:xsd=""http://www.w3.org/2001/XMLSchema"">
   <TimeSpanProperty>PT0.001S</TimeSpanProperty>
 </TypeWithTimeSpanProperty>");
-        Assert.StrictEqual(obj.TimeSpanProperty, deserializedObj.TimeSpanProperty);
+        if (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx470OrNewer())
```

Comment:
Have tried running the test against `netfx`? The test probably would still fail at the string comparison in `SerializeAndDeserialize`. We may need to skip the entire test.